main.js

```diff
@@ -0,0 +1,97 @@
+//google maps API key AIzaSyCAAdQ7E1zXvg05JhSDuKCj0NVU89JpmpI
+
+const ApiKey = "b8f04aa7ef4156ea1c923e161e7dcda6";
+var userInput;
+var days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday']
+
+getWeather = function() {
+  var current;
+  var fiveDay = [{},{},{},{},{}];
```

Comment:
Better! I at least know this will be 5 days worth of weather data and will be represented by objects